ts/webui/src/static/model/trial.ts

```diff
@@ -27,13 +28,14 @@ import {
  * @returns Parsed structured parameters and unexpected entries
  */
 function inferTrialParameters(
-    paramObj: object,
+    paramObj: RetiariiParameter,
```

Comment:
Why is this `RetiariiParameter`? How about normal trials?

Reply:
@Lijiaoa

Reply:
I think the changes above are not needed. In L35, we can put:

```typescript
'mutation_summary' in paramObj ? (paramObj as any).mutation_summary : paramObj;
```

Reply:
@ultmaster @QuanluZhang  also fix this in PR #4314